app/services/attribute_asserter.rb

```diff
@@ -66,7 +66,7 @@ def attribute_getter_function(attr)
   end
 
   def attribute_getter_function_ascii(attr)
-    ->(_principal) { decrypted_pii[attr].ascii }
+    ->(_principal) { decrypted_pii[attr].to_ascii }
```

Comment:
I believe the `to_ascii` method comes from the Stringex gem, which is not `required` in the Gemfile. I'm surprised the tests are passing without requiring stringex in this file. Could it be that because it's `required` in `app/services/idv/session.rb`, and the way Rails loads files within `app`, it automatically becomes available to other files?

Reply:
Looking into this further, it looks like this was originally added with the assumption that some Service Providers would want attributes returned to them in ASCII, but I could not find any reference to a specific SP requesting this. From the original issue that was opened (https://github.com/18F/identity-private/issues/1448), it looks like this was for SAML agencies only, but we don't have any SAML agencies right now. Furthermore, our documentation does not mention this `ascii` attribute: https://developers.login.gov/attributes/

I vote for removing any code related to ascii and the Stringex gem.

Reply:
sounds good, created https://cm-jira.usa.gov/browse/LG-274